SECURITY.md

```diff
@@ -0,0 +1,42 @@
+# Security Policies and Procedures
+
+This document outlines security procedures and general policies for the CyberArk Conjur
+project.
+
+  * [Reporting a Bug](#reporting-a-bug)
+  * [Disclosure Policy](#disclosure-policy)
+  * [Comments on this Policy](#comments-on-this-policy)
+
+## Reporting a Bug
+
+The CyberArk Conjur team and community take all security bugs in Conjur seriously.
+Thank you for improving the security of Conjur. We appreciate your efforts and
+responsible disclosure and will make every effort to acknowledge your
+contributions.
+
+Report security bugs by emailing the lead maintainers at security@conjur.org.
+
+The maintainers will acknowledge your email within 2 business days, and will send a
+more detailed response within 2 business days indicating the next steps in handling
+your report. After the initial reply to your report, the security team will
+endeavor to keep you informed of the progress towards a fix and full
+announcement, and may ask for additional information or guidance.
+
+Report security bugs in third-party modules to the person or team maintaining
+the module.
+
+## Disclosure Policy
+
+When the security team receives a security bug report, they will assign it to a
+primary handler. This person will coordinate the fix and release process,
+involving the following steps:
+
+  * Confirm the problem and determine the affected versions.
+  * Audit code to find any potential similar problems.
+  * Prepare fixes for all releases still under maintenance. These fixes will be
+    released as fast as possible.
+
```

Comment:
It seems like there's a few more steps here - we're not actually disclosing anything in this section. Do we need to include things like getting a CVE number, registering the vulnerability with GitHub, communicating to our customers the need to upgrade, and so on? Maybe this is contextual enough that the step is just something like `* Communicate the vulnerability publicly through the appropriate channels as determined by the vulnerability`.

```suggestion
  * Communicate the vulnerability and the need to upgrade to a fixed version publicly in appropriate channels
```

Reply:
I know that we are working on defining more clearly what the next steps are for communication in a situation like this, and I definitely agree we should update this file to be absolutely clear to community members what that policy is once we've defined it. But for now, I think adding this comment is premature and we should hold off.